src/main/java/org/wso2/carbon/module/swiftiso20022/MT940FormatValidator.java

```diff
@@ -98,30 +98,30 @@ public void connect(MessageContext messageContext) {
      * Method to validate the MT940 format.
      *
      * @param messageContext  Message context
-     * @param lines           Payload lines
-     * @return                Error model
+     * @param lines           Payload lines containing the Mt940 fields
+     * @return        Validation Result with error details if there is an error, else empty Validation Result
      */
-    private ErrorModel validateMT940(MessageContext messageContext, String[] lines) {
+    private ValidationResult validateMT940(MessageContext messageContext, String[] lines) {
         Map<String, Object> extractFields = extractFields(lines);
 
         if (!MT940ValidationUtils.isValidC1Rule(lines)) {
-            return new ErrorModel(ConnectorConstants.ERROR_C24,
+            return new ValidationResult(ConnectorConstants.ERROR_C24,
                     ConnectorConstants.ERROR_FIELD_86);
         }
 
         if (!MT940ValidationUtils.isValidC2Rule(lines)) {
-            return new ErrorModel(ConnectorConstants.ERROR_C27,
+            return new ValidationResult(ConnectorConstants.ERROR_C27,
                     ConnectorConstants.ERROR_BALANCES);
         }
 
-        ErrorModel errorModel = MT940ValidationUtils.validateMT940Format(extractFields);
-        if (errorModel.isError()) {
+        ValidationResult errorModel = MT940ValidationUtils.validateMT940Format(extractFields);
```

Comment:
shall we change the variable name 'errorModel' to match the type change?

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/7f2058c938da71ec80755d296fd2d3d67541e678